tests/conftest.py

```diff
@@ -410,7 +413,9 @@ def finalizer():
                 instance.ocp.wait_for_delete(
                     instance.name
                 )
-
+                if instance.kind == constants.PVC:
+                    if instance.backed_reclaim_policy == constants.RECLAIM_POLICY_DELETE:
+                        helpers.validate_pv_delete(instance.backed_pv)
```

Comment:
Could you please improve validate_pv_delete() with a proper assertionErro message?

Reply:
Updated!